Sources/momento/CacheClient.swift

```diff
@@ -0,0 +1,154 @@
+import Foundation
+
+typealias CacheClientProtocol = ControlClientProtocol & DataClientProtocol
+
+@available(macOS 10.15, iOS 13, *)
+public class CacheClient: CacheClientProtocol {
+    private let credentialProvider: CredentialProviderProtocol
+    private let configuration: CacheClientConfigurationProtocol
+    private let controlClient: ControlClientProtocol
+    private let dataClient: DataClientProtocol
+    
+    init(
+        configuration: CacheClientConfigurationProtocol,
+        credentialProvider: CredentialProviderProtocol,
+        defaultTtlMillis: TimeInterval
```

Comment:
Looks like `TimeInterval` is in seconds not in millis: https://developer.apple.com/documentation/foundation/timeinterval